pkg/operator/sync.go

```diff
@@ -446,7 +446,7 @@ func newContainers(config *OperatorConfig, features map[string]bool) []corev1.Co
 		"--logtostderr=true",
 		"--v=3",
 		"--leader-elect=true",
-		"--leader-elect-lease-duration=90s",
+		"--leader-elect-lease-duration=120s",
```

Comment:
this is the only confusing part to me. we set the lease duration in the controller-runtime config options, do we also need to set on the command line?

Reply:
We use the default if not specified on the CLI.  This is useful for development/debugging.

Reply:
ahh, perfect. thanks for the explanation Mike!